### PR TITLE
Overhaul

### PR DESCRIPTION
**Pull request by leonweber**
- Fixed discrepancies between text and summary table. I ommitted the obvious results (provability of T3  and counter-satisfiability of MC) and I also didn't mention the counter-satisfiability of A2' in AOE'. Do you think we should mention the last one?
- Added figure depicting Anderson's simplification. Here I am not sure which naming convention would apply to the definition of positive. What do you think?
-  Moved definitions from footnotes into text. This may need some further shifting around for cosmetic reasons.

### CODE REVIEW

File: Papers/InProgress/HajekAndersonControversy/main.tex [hunk 1]
```diff
@@ -302,24 +307,22 @@
...
 \begin{minipage}{.94\columnwidth}\small
 \begin{itemize}
-\item[H:A12] The negation of a property necessarily implied by a
-  positive property is not positive:
-  $$\hol{\allq \varphi \allq \psi [(P(\varphi) \wedge \nec \allq x [\varphi(x)
-  \imp \psi(x)]) \imp \neg P(\neg \psi)]}$$
-\item[H:D1] A \emph{God-like} being necessarily possesses those and only those properties that are necessarily implied by a positive property:
-  $$\hol{G_H(x) \equiv \forall \varphi [\nec \varphi(x) \biimp \exq \psi[P(\psi) \wedge \nec \allq x [\psi(x)  \imp \varphi(x))]]}$$
+\item[H:DP] A \emph{God-like} being necessarily possesses those and only those properties that are necessarily implied by a positive property:
+  $$\hol{P(x) \equiv \forall \varphi [\nec \varphi(x) \biimp \exq \psi[P(\psi) \wedge \nec \allq x [\psi(x)  \imp \varphi(x))]]}$$
```
Comment (Ceilican) [inline Papers/InProgress/HajekAndersonControversy/main.tex]: I do not understand this change (replacement of H:A12 and H:D1 by H:DP). I do not remember seeing this H:DP before. Moreover, H:A12 and H:D1 seem correct according to AOE' in section 4 of Hajek 2002 and in the Isabelle file.  Why did you do this replacement, Leon?  I will revert it for now.

Comment (Ceilican) [inline Papers/InProgress/HajekAndersonControversy/main.tex]: I will revert it, but please tell me if you think I am mistaken.

**Comment by leonweber** [comment]
My changes to AOE' make no sense whatsoever. I was playing around with the naming scheme (DP = Definition Positive) and probably altered the wrong figure. Your revert is perfectly fine. Sorry about that.

**Comment by Ceilican** [comment]
No problem.
In case you use Mac, I would suggest using the "SourceTree" git client. It allows you to visualize your changes before committing, thus minimizing the chance of non-sensical commits. There should be similar clients for Linux and Windows as well.

**Pull request merged**